add-read-me (#5)
aziontech/azion-samples: README.md

# azion-samples

> This repository presents a variety of **code samples** in JavaScript to help developers to get familiar with coding edge functions and have them running at the edge, using Azion Edge Platform.

Reviewer: ```suggestion
This repository presents a variety of **code samples** in JavaScript to help developers get familiar with coding edge functions and have them running at the edge with the Azion Edge Platform.
```

The functions are written in JavaScript and represent the approach to be taken to run them through the [Azion Edge Runtime](https://www.azion.com/en/documentation/products/edge-application/edge-functions/runtime/overview/).

## Edge functions 

> [Azion Edge Functions](https://www.azion.com/en/documentation/products/edge-application/edge-functions/) allows you to create event-driven, serverless applications, at the edge of the network, closer to users.

Reviewer: ```suggestion
[Azion Edge Functions](https://www.azion.com/en/documentation/products/edge-application/edge-functions/) allow you to create event-driven, serverless applications, at the edge of the network, closer to users.
```

> With Edge Functions, you can perform serverless functions in response to events on edge nodes of our network with no need of having or managing servers.

Reviewer: ```suggestion
With Edge Functions, you can perform serverless functions in response to events on the edge nodes of Azion's network with no need of having or managing servers.
```

## Edge functions on Edge Firewall

Through the [edge functions on Edge Firewall](https://www.azion.com/en/documentation/products/edge-firewall/edge-functions/firewall/), you can write your own security source code in JavaScript and deploy it to run at the edge of the network, as well as:

- Boost your protection.
- Have more dynamism.
- Apply the logic that your business requires.
- Use APIs that provide request and response headers manipulation.

> ## About the samples

Reviewer: ```suggestion
## About the code samples
```

# azion-samples

> This repository presents a variety of code **samples** in JavaScript to help developers to get familiar with coding edge functions and have them
> running at the edge on the Azion Edge Platform.

Reviewer 2: ```suggestion
This repository presents a variety of **code samples** in JavaScript to help developers to get familiar with coding edge functions and have them running at the edge, using Azion Edge Platform.
```

The functions are written in JavaScript and represent the approach to be taken to run them through the [Azion Edge Runtime](https://www.azion.com/en/documentation/products/edge-application/edge-functions/runtime/overview/).

## Edge functions 

> [Azion Edge Functions](https://www.azion.com/en/documentation/products/edge-application/edge-functions/) allows you to create event-driven, serverless applications, at the edge of the network, closer to users.

Reviewer 2: ```suggestion
[Azion Edge Functions]({% tl documentation_products_edge_functions%}) module allows you to create event-driven, serverless applications, at the edge of the network, closer to users.
```

Reviewer 2: adicionei o "module" para ficar melhor a correspondência com o verbo em 3a pessoa.

Author: nesse caso n vai funcionar o namespace, pq é somente aqui no github que vai aparecerrr

With Edge Functions, you can perform serverless functions in response to events on edge nodes of our network with no need of having or managing servers.

## Edge functions on Edge Firewall

> Through the [edge functions on Edge Firewall](https://www.azion.com/en/documentation/products/edge-firewall/edge-functions/firewall/) you can write your own security source code in JavaScript and deploy it to run at the edge of the network, as well as:

Reviewer 2: ```suggestion
Through the [edge functions on Edge Firewall](https://www.azion.com/en/documentation/products/edge-firewall/edge-functions/firewall/), you can write your own security source code in JavaScript and deploy it to run at the edge of the network, as well as:
```

- Boost your protection.
- Have more dynamism.
- Apply the logic that your business requires.
- Use APIs that provide request and response headers manipulation.

## About the samples

> The functions in this directory were created as examples to be followed, so based on them, you can apply the logic your application requires.

Reviewer 2: ```suggestion
The functions in this directory were created as examples to be followed. Based on them, you can apply the logic your application requires.
```

> **Move to the Edge**!!!

Reviewer 2: ```suggestion
**Move to the Edge**
```

Reviewer 2: a recomendação é evitar !!

Author: mesmo que nao seja no portal ??
